GetPopularAssetTrigger/index.ts

```diff
@@ -44,7 +43,7 @@ const GetPopularAssets: AzureFunction = async function (context: Context, req: H
     TE.chain(({ approvedStatus, dataProductType }) =>
       TE.tryCatch(
         () => getMostViewedDataProducts(collibraClient)([], limit, 0, approvedStatus.id, dataProductType.id),
-        (err: AxiosError) => toNetErr(err.response.status ?? 500)(err.message)
+        (err: any) => toNetErr(err.response.status ?? 500)(err.message)
```

Comment:
i suggest we update the `toNetErr` implementation instead of typing `err` as `any`.

the new implementation should take in an error of `unknown` type and return a `NetError` instance.

you can look at the [E.toError](https://github.com/gcanti/fp-ts/blob/master/src/Either.ts#L1401-L1403) implementation for inspiration